style/ruby/README.md

```diff
@@ -25,6 +24,7 @@ Ruby
 * Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
   calls.
 * Prefer `if` over `unless`.
+* Prefer explicit, early `return`s over `if`-`else`.
```

Comment:
At the very least, if we're in keeping with all of the reading around guard clauses, I think it would be prudent for this addition to suggest preferring guard clauses instead of nested conditionals, rather than preferring guard clauses over any conditional, which I don't agree with.